kubectl-fdb/cmd/suite_test.go

```diff
@@ -51,3 +41,17 @@ var _ = JustBeforeEach(func() {
 var _ = AfterEach(func() {
 	k8sClient.Clear()
 })
+
+func createCluster(givenName string, givenNamespace string) *fdbv1beta2.FoundationDBCluster {
```

Comment:
```suggestion
func createCluster(name string, namespace string) *fdbv1beta2.FoundationDBCluster {
```

Do you think the prefix "given" adds additional value to this parameters?

The name `createCluster` is a bit misleading as we are not really creating a cluster, we only generate the cluster struct (e.g. the `createPods` actually "creates" those Pods in the Kubernetes instance).